webpack/assets/javascripts/react_app/components/UserJwtToken/UserJwtTokenSelectors.js

```diff
@@ -0,0 +1,11 @@
+import { USER_JWT_TOKEN } from './UserJwtTokenConstants';
+import {
+  selectAPIByKey,
+  selectAPIResponse,
+} from '../../redux/API/APISelectors';
+
+export const selectUserJwtToken = state =>
+  selectAPIResponse(state, USER_JWT_TOKEN).jwt_token || null;
+
+export const selectStatus = state =>
+  selectAPIByKey(state, USER_JWT_TOKEN).status || null;
```

Comment:
I can't use `selectAPIStatus(state, USER_JWT_TOKEN)` because it returns by default `PENDING` value.
Don't know why though, it should be `NULL` since no request has been made IMO.

Reply:
nice catch! opened a small fix for it https://github.com/theforeman/foreman/pull/7956

Reply:
..And now it got merged :)